FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -4,9 +4,20 @@
 
 import Foundation
 
+public struct FeedImageListRemote: Decodable {
+	public let items: [FeedImageRemote]
+}
+
+public struct FeedImageRemote: Decodable {
+	public let imageId: UUID
+	public let imageDesc: String?
+	public let imageLoc: String?
+	public let imageUrl: URL
```

Comment:
Since `FeedImageRemote ` should be private, no need to make its properties public, please remove `public` modifiers

```suggestion
	let imageId: UUID
	let imageDesc: String?
	let imageLoc: String?
	let imageUrl: URL
```